blocks/library/paragraph/index.js

```diff
@@ -117,21 +144,63 @@ class ParagraphBlock extends Component {
 			),
 			isSelected && (
 				<InspectorControls key="inspector">
-					<PanelBody title={ __( 'Text Settings' ) }>
+					<PanelBody title={ __( 'Text Settings' ) } className="blocks-font-size">
```

Comment:
I expect we're planning to extract this to a separate component?

Reply:
Indeed.